vars/getFailureMessage.groovy

```diff
@@ -55,20 +55,28 @@ def formatForJSON(String message) {
 
 def getTestFailures(String log) {
 	def maxStackTraceLength = 500
-	def foundFailures = false
+	def failuresFound = 0
 	def failedTestsMessage = "<h4>Failed Tests:</h4> <ul>"
 	def failedTestPattern = Pattern.compile("[a-z].*<<< (FAILURE|ERROR)!\n")
 	def failedTestMatcher = failedTestPattern.matcher(log)
+	def maxFailureNum = 25
 	while(failedTestMatcher.find()) {

```

Comment:
```suggestion
	while(failedTestMatcher.find() && ++failuresFound <= maxFailureNum) {
```

Reply:
I actually want the loop to run through all the test failures to count them. 
Could commit this suggestion and then modify the regex at L74 to have capturing groups and get the # of test failures that way but this works and its less complicated.

Reply:
my mistake